test/runtests.jl

```diff
@@ -10,7 +10,7 @@ testnames = [
     "floatapprox", "readdlm", "reflection", "regex", "float16", "combinatorics",
     "sysinfo", "rounding", "ranges", "mod2pi", "euler", "show",
     "lineedit", "replcompletions", "repl", "replutil", "sets", "test", "goto",
-    "llvmcall", "grisu", "nullable", "meta", "profile",
+    "llvmcall", "grisu", "nullable", "meta", "profile", "ccall",
```

Comment:
Won't this make `Base.runtests` fail in any binary install where `libccalltest` won't be present?